framework/src/media/utils/MediaUtils.cpp

```diff
@@ -511,6 +583,51 @@ bool header_parsing(unsigned char *buffer, unsigned int bufferSize, audio_type_t
 	return true;
 }
 
+bool ts_parsing(unsigned char *buffer, unsigned int bufferSize, audio_type_t *audioType, unsigned int *channel, unsigned int *sampleRate, audio_format_type_t *pcmFormat)
+{
+#ifdef CONFIG_MPEG2_TS
```

Comment:
This function is being used only for MPEG TS then please wrap up whole of function itself with config.

Reply:
fixed